src/renderer/dx/CustomTextRenderer.cpp

```diff
@@ -635,6 +650,22 @@ CATCH_RETURN()
 }
 #pragma endregion
 
+[[nodiscard]] HRESULT CustomTextRenderer::EndFrame(void* clientDrawingContext) noexcept
+try
+{
+    DrawingContext* drawingContext = static_cast<DrawingContext*>(clientDrawingContext);
+    RETURN_HR_IF(E_INVALIDARG, !drawingContext);
+
+    if (_hasClipPushed)
+    {
+        drawingContext->renderTarget->PopAxisAlignedClip();

```

Comment:
Concern: popping the clip but not clearing the clip location could result in the next run through `DrawGlyphRun` not Pushing a clip _if it would be the same_

Reply:
I expect that this will not be hit in practice because DrawGlyphRun will not be called back to back with the same position? Still a concern.

Reply:
I'll clear it.

Reply:
Or I could just make it an optional and use the flag based on an empty optional and that'd solve it...